ui/webpack.config.js

```diff
@@ -23,6 +25,23 @@ module.exports = merge(baseConfig, {
             },
         ],
     },
-    plugins: [new LicenseWebpackPlugin(), new ForkTsCheckerWebpackPlugin()],
+    plugins: [
+        new LicenseWebpackPlugin(),
+        new ForkTsCheckerWebpackPlugin(),
+        new CopyPlugin({
+            patterns: [
+                {
+                    from: 'templates',
+                    context: 'appserver',
+                    to: '../../../template', // /dist/templates
+                },
```

Comment:
I remember that we moved those away from UI as those are not used in the UI directly, what is the reason to bring them back?

Reply:
I wanted to add hash entry_page.js to avoid caching problems, but faced with issue that this base.html is processing by python template. I will move it back.